test/constexpr_vector.ll

```diff
@@ -0,0 +1,66 @@
+; RUN: llvm-as < %s | llvm-spirv -s | llvm-dis | FileCheck %s --check-prefix=CHECK-LLVM
+
```

Comment:
Please add also checks for SPIR-V representation.
Other then that LGTM. Thanks!